app/javascript/components/close_contact/CloseContact.js

```diff
@@ -132,137 +142,151 @@ class CloseContact extends React.Component {
+              </Form.Control.Feedback>
+            </Form.Group>
+            <Form.Group as={Col} lg="12" controlId="assigned_user">
+              <Form.Label className="nav-input-label">
+                Assigned User {schema?.fields?.assigned_user?._exclusive?.required && '*'}
+                <InfoTooltip tooltipTextKey="assignedUser" location="top"></InfoTooltip>
+              </Form.Label>
+              <Form.Control
+                isInvalid={this.state.errors['assigned_user']}
+                as="input"
+                list="assigned_users"
+                autoComplete="off"
+                size="lg"
+                className="d-block"
+                onChange={this.handleChange}
+                value={this.state.assigned_user || ''}
+              />
+              <datalist id="assigned_users">
+                {this.props.assigned_users?.map(num => {
+                  return (
+                    <option value={num} key={num}>
+                      {num}
+                    </option>
+                  );
+                })}
+              </datalist>
+              <Form.Control.Feedback className="d-block" type="invalid">
+                {this.state.errors['assigned_user']}
+              </Form.Control.Feedback>
+            </Form.Group>
+          </Row>
+          <Row>
+            <Form.Group as={Col}>
+              <Form.Label htmlFor="notes" className="nav-input-label">
+                Notes
+                {schema?.fields?.notes?._exclusive?.required && '*'}
+              </Form.Label>
+              <Form.Control
+                id="notes"
+                as="textarea"
+                rows="5"
+                className="form-square"
+                value={this.state.notes || ''}
+                placeholder={this.closeContactNotePlaceholder}
+                maxLength="2000"
+                onChange={this.handleChange}
+              />
+              <Form.Label className="notes-character-limit"> {2000 - this.state.notes.length} characters remaining </Form.Label>
+              <Form.Control.Feedback className="d-block" type="invalid">
+                {this.state.errors['notes']}
+              </Form.Control.Feedback>
+            </Form.Group>
+          </Row>
         </Modal.Body>
         <Modal.Footer>
           <Button variant="secondary btn-square" onClick={toggle}>
             Cancel
           </Button>
-          <Button
-            variant="primary btn-square"
-            onClick={submit}
-            disabled={
-              !(this.state.first_name || this.state.last_name || this.state.email || this.state.primary_telephone || this.state.notes) || this.state.loading
-            }>
-            {this.props.close_contact.id ? 'Update' : 'Create'}
+          <Button variant="primary btn-square" onClick={submit} disabled={this.state.disableCreate || this.state.loading}>
+            <span data-for="create-tooltip" data-tip="" className="ml-1">
```

Comment:
The little carrot for the tooltip is missing - normally indicates what the tooltip is attached too.  Can we add this back somehow?

![Screen Shot 2021-02-04 at 8 41 04 AM](https://user-images.githubusercontent.com/35042815/106900644-c0406180-66c4-11eb-9db8-edc3191ea152.png)

Reply:
Well yes and no. No you cant have the arrow offset from the center. But it sounds like we'll be reverting the tooltip to as close to default as possible, so that solves the problem altogether